### ISSUE TITLE
photoreceptor cell death.

### ISSUE THREAD

**Issue by LOURGHI** [obophenotype/human-phenotype-ontology#1219]
Please consider a new term for photoreceptor cell death .
Stargardt disease,PMID:27491360.
Related synonym to Photoreceptor layer loss on macular OCT???

**Comment by psergouniotis** [comment]
Photoreceptor cell death is a histological term. In the above paper the authors refer to [Photoreceptor layer loss on macular OCT HP:0030609] which suggests photoreceptor cell death/degeneration.

**Comment by pnrobinson** [comment]
Although this may not be a common measurement, I have found this
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4600048/
Adaptive optics scanning light ophthalmoscopy (AOSLO) enables visualization of single photoreceptor cells in the living human retina, including foveal cones and peripheral rods. Diseased photoreceptors can be measured quantitatively, and cone loss has been shown to correlate with clinically identified lesions in STGD1. To investigate early STGD1, we used AOSLO to evaluate 2 brothers with macular atrophy, a mild phenotype, and their unaffected parents.

Therefore, it is possible to measure this feature clinically. I would not use the term "PHotoreceptor cell death, but instead simply "Reduced photoreceptor density"
